spec/requests/citizens/property_value_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Citizens::PropertyValuesController, type: :request do
+  describe 'citizen property value test' do
+    describe 'GET /citizens/property_value' do
+      before { get citizens_property_value_path }
+
+      it 'returns http success' do
+        expect(response).to have_http_status(:ok)
+        expect(unescaped_response_body).to include('How much is your home worth')
```

Comment:
A pedantic point, but you don't need to use `unescaped_response_body` if you know the text you are matching does not contain any problematic characters. As you know the matching text is `'How much is your home worth'` So `response.body` would be fine here.